oracle/README.md

```diff
@@ -12,11 +12,9 @@ Get metrics from Oracle Database servers in real time to visualize and monitor a
 
 #### Prerequisite
 
-To use the Oracle integration you can either use the native client (no additional install steps required) or download the Oracle JDBC driver (Linux only). To use the Oracle integration with JDBC, download the Oracle JDBC driver. If not using the JDBC method, the minimum [supported version][2] is Oracle 12c.
+To use the Oracle integration you can either use the native client (no additional install steps required), the Oracle Instant Client, or download the Oracle JDBC driver (Linux only). To use the Oracle integration with JDBC, download the Oracle JDBC driver. If not using the JDBC method, the minimum [supported version][2] is Oracle 12c.
```

Comment:
```suggestion
To use the Oracle integration you can either use the native client (no additional install steps required), the Oracle Instant Client via `python-oracledb` Thick mode, or download the Oracle JDBC driver (Linux only). To use the Oracle integration with JDBC, download the Oracle JDBC driver. If not using the JDBC method, the minimum [supported version][2] is Oracle 12c.
```
Just adding this note since you mention `Thick mode` later in this README but I don't think we mention what `Thick mode` refers to.